repose-aggregator/components/services/atom-feed-service/atom-feed-service-impl/src/main/scala/org/openrepose/nodeservice/atomfeed/impl/auth/OpenStackIdentityV2AuthenticatedRequestFactory.scala

```diff
@@ -44,7 +45,7 @@ object OpenStackIdentityV2AuthenticatedRequestFactory {
 /**
   * Fetches a token from the OpenStack Identity service, if necessary, then adds the token to the request.
   */
-class OpenStackIdentityV2AuthenticatedRequestFactory(configuration: OpenStackIdentityV2AuthenticationType)
+class OpenStackIdentityV2AuthenticatedRequestFactory @Inject()(configuration: OpenStackIdentityV2AuthenticationType)
```

Comment:
This was a bit more of a pain than I had anticipated when you mentioned it. I think it is mostly doing it, but there are a couple of tests I think I broke along the way. I will fix them after the Jenkins run.